internal/backends/sqlite/database.go

```diff
@@ -169,8 +169,8 @@ func (db *database) Stats(ctx context.Context, params *backends.DatabaseStatsPar
 	}, nil
 }
 
-// relationStats returns statistics about tables and indexes for the given collections.
-func relationStats(ctx context.Context, db *fsql.DB, list []*metadata.Collection) (*stats, error) {
+// collectionsStats returns statistics about tables and indexes for the given collections.
```

Comment:
@AlekSi then suggest a better wording, please. If I say "returns statistics about table and indexes for the given collections" it will sound like there is only one table for all the given collections.